### PR TITLE
Update deprecated qos policy value names

### PR DESCRIPTION
**Pull request by ivanpauno**
See ros2/rclpy#634.

### CODE REVIEW

File: ros2topic/test/test_cli.py [hunk 1]
```diff
@@ -295,11 +295,11 @@ def test_topic_endpoint_info_verbose(self):
                 re.compile(r'Endpoint type: (INVALID|PUBLISHER|SUBSCRIPTION)'),
                 re.compile(r'GID: [\w\.]+'),
                 'QoS profile:',
-                re.compile(r'  Reliability: RMW_QOS_POLICY_RELIABILITY_\w+'),
-                re.compile(r'  Durability: RMW_QOS_POLICY_DURABILITY_\w+'),
+                re.compile(r'  Reliability: (RELIABLE|BEST_EFFORT|SYSTEM_DEFAULT|UNKNOWN)'),
+                re.compile(r'  Durability: (VOLATILE|TRANSIENT_LOCALE|SYSTEM_DEFAULT|UNKNOWN)'),
```
Comment (jacobperron) [inline ros2topic/test/test_cli.py:299]:
```suggestion
                re.compile(r'  Durability: (VOLATILE|TRANSIENT_LOCAL|SYSTEM_DEFAULT|UNKNOWN)'),
```

**Review by jacobperron — COMMENTED** [review]
One typo.

**Review by jacobperron — APPROVED** [review]

**Comment by jacobperron** [comment]
@ros-pull-request-builder retest this please

**Comment by ivanpauno** [comment]
Rpr checker will be fixed when I make a new rclpy release (which I will do today).

**Pull request merged**